frontend/src/app/studies/studies.component.html

```diff
@@ -1,37 +1,51 @@
 <div
-  class="studies"
-  fxLayout.lt-md="column"
-  fxLayout.gt-sm="row"
-  fxLayoutGap="40px"
-  fxLayoutAlign="center auto"
+  id="hero"
+  class="center-center gradient-overlay container with-escaping-tiles"
 >
-  <div fxFlex.gt-sm="30%">
-    <h1>STAR Research</h1>
-    <h2>About our research</h2>
-    <p>The Supporting Transformative Autism Research (STAR) project was designed to
-      ignite discovery, improve systems of care, and equip communities with best practices
-      to support people with autism across the lifespan. STAR brings together researchers,
-      community partners, individuals with autism and their families to identify challenges
-      and co-develop innovative solutions.</p>
-    <p>We aim to address critical knowledge gaps in areas that have the greatest potential
-      to improve the lives of individuals with autism and their families.  Our projects focus
-      on understanding the biological, social and cognitive underpinnings of autism,
-      developing more effective and personalized interventions and supports, and developing
-      innovative models to bridge the gap between research and practice through training and
-      knowledge sharing.  Our research ranges from basic neuroscience to applied practice.
-      The ultimate goal of this work is to empower individuals with autism, their families,
-      and their communities to achieve positive outcomes and quality of life.</p>
+  <div class="hero-content">
+    <h1>Autism Research Studies</h1>
   </div>
   <div
-    fxFlex.gt-sm="60%"
-    fxLayout="column"
+    class="row escaping-tiles-1x"
+    fxLayout="row"
     fxLayoutGap="40px"
-    *ngIf="query && query.hits"
+    fxLayoutAlign="center auto"
+    id="TopOfSearch"
```

Comment:
This page is looking really nice now. I think the team should be very pleased. They were worried about the page lacking intention/focus, and I think this really solves that.